browser/ui/views/frame/brave_contents_view_util.cc

```diff
@@ -12,10 +12,9 @@ namespace {
 
 constexpr ViewShadow::ShadowParameters kShadow{
     .offset_x = 0,
-    .offset_y = 1,
-    .blur_radius = 4,
-    .shadow_color = SkColorSetA(SK_ColorBLACK, 0.07 * 255)};
-
+    .offset_y = 0,
```

Comment:
With the `offset_y` set the shadow between the location bar and the web contents was hard to see, so I removed it